src/satosa/satosa_config.py

```diff
@@ -76,12 +101,14 @@ def _verify_dict(self, conf):
         :param conf: config to verify
         :return: None
         """
-        if not conf:
-            raise SATOSAConfigurationError("Missing configuration or unknown format")
 
         for key in SATOSAConfig.mandatory_dict_keys:
-            if key not in conf:
-                raise SATOSAConfigurationError("Missing key '%s' in config" % key)
+            if not conf.get(key, None):
+                raise SATOSAConfigurationError(
+                    "Missing key {key} or value for {key} in config".format(
+                        key=key
+                    )
+                )
```

Comment:
If I have three keys missing, should I have to try three times to get the complete information?
Instead, can we turn this into an actual validator?